CHANGELOG.md

```diff
@@ -7,6 +7,13 @@ About changelog [here](https://keepachangelog.com/en/1.0.0/)
 
 ## [x.x.x]
```

Comment:
It will come back and haunt you with the first new PR.. ;-)

Seriously, well done!

Reply:
Did I forget the date?

Reply:
No, just as usual added the new changelog entry on version bump, and not immediately after. A reasonable optimisation that sometimes causes confusion on the first merge to the new development version.

Reply:
I wasn't confused about it! ;)

Reply:
😆 Of course not.